docs/intro/import.mdx

```diff
@@ -12,99 +12,124 @@ You can use any of the three following ways to get ValQ added to Microsoft Power
 2.  Import from AppSource - Marketplace
 3.  Import from AppSource - My Organization
 
+<div style={{textAlign: 'center'}}>
 <img alt="ValQ get" src={useBaseUrl('/doc-images/intro/install1.png')} 
     width={430}
-    height={395}
-/>
+    height={395}/>
+</div>
 
 ## 1. Import from file
```

Comment:
Can we move this section as the last section? 
1. App Source
2. My Organization
3. File